cmd/mailchain/internal/http/handlers/send_message.go

```diff
@@ -94,6 +93,10 @@ func SendMessage(sent stores.Sent, senders map[string]sender.Message, ks keystor
 			errs.JSONWriter(w, http.StatusUnprocessableEntity, errors.WithStack(errors.WithMessage(err, "could not get `signer`")))
 			return
 		}
+		encrypter, err := encrypter.GetEncrypter(req.Body.Encryption)
+		if err != nil {
+			errs.JSONWriter(w, http.StatusUnprocessableEntity, errors.WithStack(errors.WithMessage(err, "could not get `encrypter`")))
```

Comment:
`errs.JSONWriter(w, http.StatusUnprocessableEntity, errors.WithMessage(err, "could not get `encrypter`"))` is enough. `errors.WithMessage` will also include the stack information.